Marsiple/App/Post Details/PostDetailsViewController.swift

```diff
@@ -48,15 +48,23 @@ extension PostDetailsViewController: UITableViewDataSource {
     }
 }
 
+private extension PostDetailsViewController {
+    @objc func commentButtonTapped() {
+        let commentView = CommentViewController()
+        navigationController?.pushViewController(commentView, animated: true)
+    }
+}
+
 private extension PostDetailsViewController {
     func setupNavigationBar() {
         navigationItem.title = LocalizationKey.PostDetails.navigationBarTitle.localized()
+        let commentButton = UIBarButtonItem(title: "Comment", style: .plain, target: self, action: #selector(commentButtonTapped))
```

Comment:
I think we could introduce an extension on `UIBarButtonItem` where we have `static` methods providing concrete, contextually customised `UIBarButtonItem` instances.

In example:
```
static func createCommentItem(target: Any, action: Selector) -> UIBarButtonItem {
       // TODO: - Localize
       return UIBarButtonItem(title: "Comment", style: .plain, target: target, action: action)
}
```